lib/analyzer/applications/java.ts

```diff
@@ -3,40 +3,34 @@ import * as path from "path";
 import { bufferToSha1 } from "../../buffer-utils";
 import { JarFingerprintsFact } from "../../facts";
 import { JarFingerprint } from "../types";
-import { JarBuffer, JarCoords, JarInfo } from "./types";
+import { AggregatedJars, JarBuffer, JarCoords, JarInfo } from "./types";
 import { AppDepsScanResultWithoutTarget, FilePathToBuffer } from "./types";
 
+/**
+ * @param {{[fileName: string]: Buffer}} input fileName
+ * @returns {AggregatedJars}
+ */
```

Comment:
Is this a known syntax for documentation? I'm pretty new to Type Script, interesting to know!
I'm guessing you only added it here because this is the main function who call all other helper functions in the file?

Reply:
The addition is similar to existing documentation in the file and project

Reply:
But did you add it just for this function?

Reply:
Yes, I thought it would be more readable but maybe if there is no additional usage should I remove it?

Reply:
I was just wondering about why you added it just to this function. It's no big deal. You can go ahead and merge!